parse-config.go

```diff
@@ -111,6 +112,7 @@ var SubSystems = set.CreateStringSet(
 	NotifyWebhookSubSys,
 	LambdaWebhookSubSys,
 	BrowserSubSys,
+	AuditEventQueueSubSys,
```

Comment:
create a new variable

```
EOSSubsystems
```

and then add previous Subsystems and this new one, otherwise upstream pulling this would automatically show up this subsystem incorrectly.

Reply:
gotcha, will refactor